enhancedTable/panel-manager.ts

```diff
@@ -90,77 +162,74 @@ export class PanelManager {
         this.angularHeader();
 
         const menuBtn = new this.panel.container(MENU_TEMPLATE);
-        menuBtn.element.css('float', 'right');
 
         const closeBtn = new this.panel.button('X');
-        closeBtn.element.css('float', 'right');
 
         const searchBar = new this.panel.container(SEARCH_TEMPLATE);
-        searchBar.element.css('float', 'right');
 
         const clearFiltersBtn = new this.panel.container(CLEAR_FILTERS_TEMPLATE);
-        clearFiltersBtn.element.css('float', 'right');
 
         const columnVisibilityMenuBtn = new this.panel.container(COLUMN_VISIBILITY_MENU_TEMPLATE);
-        columnVisibilityMenuBtn.element.css('float', 'right');
 
-        return [closeBtn, menuBtn, clearFiltersBtn, columnVisibilityMenuBtn, searchBar];
+        return [searchBar, columnVisibilityMenuBtn, clearFiltersBtn, menuBtn, closeBtn];
     }
 
     angularHeader() {
         const that = this;
-        this.mapApi.agControllerRegister('SearchCtrl', function() {
+        this.mapApi.agControllerRegister('SearchCtrl', function () {
```

Comment:
space between `function` and `()`. Looks like it's vs code's auto formatting. What's the style we want to use for this?